assignments/array-methods.js

```diff
@@ -55,22 +55,31 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
-let fullName = [];
-console.log(fullName);
+let fullName = []; 
+    fullName = runners.map((name) => `${name.first_name} ${name.last_name}`);
+console.log(fullName) 
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+    allCaps = runners.map((name) => name.first_name.toUpperCase())
 console.log(allCaps); 
 
 // ==== Challenge 3: Use .filter() ====
 // The large shirts won't be available for the event due to an ordering issue.  Get a list of runners with large sized shirts so they can choose a different size. Return an array named largeShirts that contains information about the runners that have a shirt size of L and log the result
 let largeShirts = [];
+largeShirts = runners.filter((size) => {
+    return size.shirt_size == "L"
+})
 console.log(largeShirts);
 
+
 // ==== Challenge 4: Use .reduce() ====
 // The donations need to be tallied up and reported for tax purposes. Add up all the donations into a ticketPriceTotal array and log the result
 let ticketPriceTotal = [];
+    ticketPriceTotal = runners.reduce((acc, currentVal) => {
```

Comment:
If you wanted to, you could assign acc the value of donation and currentVal and be assigned any name that makes sense. 

Ex:
ticketPriceTotal = runners.reduce((donation, runners) => {
    return donation += runners.donation
}, 0)
console.log('Challenge 4:', ticketPriceTotal);